proto/celestia/core/v1/blob/blob.proto

```diff
@@ -1,7 +1,7 @@
 syntax = "proto3";
 package celestia.core.v1.blob;
 
-option go_package = "github.com/celestiaorg/go-square/blob";
+option go_package = "github.com/celestiaorg/go-square/v2/share";
```

Comment:
[question] why move the Go package?

Reply:
It's probably not necessary either way because this file will be deleted in a follow up PR

Reply:
I ended up removing the file now because it's not used